java/common/common-task/src/test/java/sleeper/task/common/RunCompactionTasksTest.java

```diff
@@ -197,7 +236,7 @@ void shouldDoNothingWhenExistingTasksAreOverTarget() {
             runToMeetTargetTasks(2, existingTasks(3));
 
             // Then
-            assertThat(scaleToHostsRequests).isEmpty();
+            assertThat(scaleToHostsRequests).containsExactly(2);
```

Comment:
It looks like we can't scale down to 2 when there are 3 tasks already running? Should this scale to 3?